backend/src/docs/v1.api-spec.yaml

```diff
@@ -362,7 +362,7 @@ components:
             type: string
             example: CHES
           description: An array of applicationAcronyms
-        keycloakEnvironment:
+        clientEnvironment:
```

Comment:
Reorder this property to be before commonServices (alphabetical ordering)

Reply:
Fixed